src/theme/components/OpenPositions.js

```diff
@@ -43,51 +51,22 @@ function OpenPositions() {
         </div>
       </div>
       <div className={clsx(styles.container)}>
-        <div className={clsx(styles.positions)}>
-          <Position
-            title="Cryptographer"
-            time="Full-Time"
-            location="SF / Remote"
-            link="https://jobs.lever.co/ironfish/8c384a79-5ffe-4b27-9387-46e1410db067"
-          />
-          <Position
-            title="Engineering Manager"
-            time="Full-Time"
-            location="SF / Remote"
-            link="https://jobs.lever.co/ironfish/35a389f1-0883-455b-8c19-b9e20a936e2b"
-          />
-          <Position
-            title="Front End Engineer"
-            time="Full-Time"
-            location="SF / Remote"
-            link="https://jobs.lever.co/ironfish/8c7ef6a8-a8b4-40da-a5b1-7e54853f6dd9"
-          />
-          <Position
-            title="Full Stack Engineer"
-            time="Full-Time"
-            location="SF / Remote"
-            link="https://jobs.lever.co/ironfish/76dd75e4-f8d9-44a7-9693-4462696a8ceb"
-          />
-          <Position
-            title="Software Engineer"
-            time="Full-Time"
-            location="SF / Remote"
-            link="https://jobs.lever.co/ironfish/e0b384ba-64c6-4e31-8542-8ba0d9b01797"
-          />
+        {jobData.type === 'loading' && <div className={clsx(styles.loading)}>Loading...</div>}
 
-          <Position
-            title="Executive Assistant"
-            time="Full-Time"
-            location="SF / Remote"
-            link="https://jobs.lever.co/ironfish/77d38d59-97f9-4ba2-95ff-ee8f8f1d3cc9"
-          />
-          <Position
-            title="Community Manager"
-            time="Full-Time"
-            location="SF / Remote"
-            link="https://jobs.lever.co/ironfish/21c1357b-b41e-4474-bbe8-1690b6c3ed17"
-          />
-        </div>
+        {jobData.type === 'success' &&
+          <div className={clsx(styles.positions)}>
+            {jobData.data.map((job) => <Position
+              title={job.text}
+              time={job.categories.commitment || ''}
+              location={job.categories.location || ''}
+              link={job.hostedUrl}
+            />)}
+          </div>
+        }
+
+        {jobData.type === 'error' && <div className={clsx(styles.loading)}>
+          Visit our <Link to={`https://jobs.lever.co/${LEVER_ORGANIZATION}`}>Lever page</Link> to view our job listings.
+        </div>}
       </div>
```

Comment:
It seems like we need to adjust our lever data, because half of the position's don't show as full time., and all of them show as San Francisco.

Reply:
Yeah agreed

Reply:
Ok this is updated